docs/en/latest/install-dependencies.md

```diff
@@ -150,7 +150,10 @@ nohup etcd &
 
 ```shell
 # install OpenResty, etcd and some compilation tools
-brew install openresty/brew/openresty etcd luarocks curl git
+brew install openresty/brew/openresty openresty-openssl111 etcd curl git
```

Comment:
I remember openresty-openssl111 is a dependency of openresty, will it be installed automatically?

Reply:
yep, or-openssl111 will be autoaticll installed when you install openresty